kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -50,339 +40,226 @@
 import org.slf4j.LoggerFactory;
 
 final class ProducerMetrics {
-
   private static final Logger log = LoggerFactory.getLogger(ProducerMetrics.class);
 
-  private final String fullyQualifiedRequestSensor;
-  private final String fullyQualifiedRequestSizeSensor;
-  private final String fullyQualifiedResponseSensor;
-  private final String fullyQualifiedRecordErrorSensor;
-  private final String fullyQualifiedRequestLatencySensor;
-
-  private final String jmxPrefix;
-  private final Metrics metrics;
-  private final ConcurrentMap<BeanCoordinate, ProduceMetricMBean> beansByCoordinate =
-      new ConcurrentHashMap<>();
-
-  ProducerMetrics(KafkaRestConfig config, Time time) {
-    this.metrics = new MetricsBuilder(config.getMetricsContext(), time).build();
-    this.jmxPrefix = config.getString(KafkaRestConfig.METRICS_JMX_PREFIX_CONFIG);
-    String sensorNameTemplate = jmxPrefix + ":" + ProducerMetricsRegistry.GROUP_NAME + ":%s";
-    this.fullyQualifiedRecordErrorSensor =
-        String.format(sensorNameTemplate, ProducerMetricsRegistry.RECORD_ERROR_SENSOR);
-    this.fullyQualifiedRequestSensor =
-        String.format(sensorNameTemplate, ProducerMetricsRegistry.REQUEST_SENSOR);
-    this.fullyQualifiedRequestLatencySensor =
-        String.format(sensorNameTemplate, ProducerMetricsRegistry.REQUEST_LATENCY_SENSOR);
-    this.fullyQualifiedRequestSizeSensor =
-        String.format(sensorNameTemplate, ProducerMetricsRegistry.REQUEST_SIZE_SENSOR);
-    this.fullyQualifiedResponseSensor =
-        String.format(sensorNameTemplate, ProducerMetricsRegistry.RESPONSE_SENSOR);
-    setupMetricBeans();
-  }
+  private static final String GROUP_NAME = "produce-api-metrics";
+  private static final ImmutableMap<String, String> METRIC_TAGS = ImmutableMap.of();
+  private static final int NUM_SAMPLES = 10;
+  private static final long SAMPLE_WINDOWS_MS = 10000;
```

Comment:
nit: Use `L` to switch to a `long` literal. Also I'd suggest `SAMPLE_WINDOWS...` -> `SAMPLE_WINDOW...` - the fact that there are multiple windows during the metric recording is commonly understood I think.

Reply:
Done.